macros/jinja_helpers/slugify.sql

```diff
@@ -1,5 +1,9 @@
 {% macro slugify(string) %}
 
+{% if not string %}
+{{ return(string) }}
```

Comment:
What about `return('')` instead? I think that calling `slugify(None)` would return a null instead of a blank, right?

Reply:
good catch!